### DISCUSSION TITLE
Build Error when deploying Nextjs app

### DISCUSSION THREAD

**Discussion by LaurensKersbergen** [vercel/vercel#6148] [Help]
Hey! When I'm trying to deploy my Nextjs app (see: https://github.com/LaurensKersbergen/personal-website), to Vercel.

I've got it all working locally, but when I deploy to Vercel, I keep getting the Errors below. It's a Next.js app, styled with Tailwind CSS and deployed to Vercel. I'm using [this simple setup](https://www.notion.so/Static-Blog-Template-a14c02abc8d9422cb35e4334e4ac88b7).

There seems to be something wrong with the Dropbox integration I'm using. But not sure what I'm doing wrong.

Could you maybe give me a hand? Thanks in advance!

`
08:27:50.235 | > Build error occurred
-- | --
08:27:50.235 | Error: ENOENT: no such file or directory, scandir '/vercel/workpath0/_content'
08:27:50.235 | at Object.readdirSync (fs.js:1021:3)
08:27:50.235 | at getPostSlugs (/vercel/workpath0/.next/serverless/pages/[slug].js:1945:55)
08:27:50.235 | at getAllPosts (/vercel/workpath0/.next/serverless/pages/[slug].js:1990:17)
08:27:50.236 | at getStaticPaths (/vercel/workpath0/.next/serverless/pages/[slug].js:2647:85)
08:27:50.236 | at buildStaticPaths (/vercel/workpath0/node_modules/next/dist/build/utils.js:16:86)
08:27:50.236 | at Object.isPageStatic (/vercel/workpath0/node_modules/next/dist/build/utils.js:23:555)
08:27:50.236 | at processTicksAndRejections (internal/process/task_queues.js:93:5) {
08:27:50.236 | type: 'Error',
08:27:50.236 | errno: -2,
08:27:50.236 | syscall: 'scandir',
08:27:50.236 | code: 'ENOENT',
08:27:50.236 | path: '/vercel/workpath0/_content'
08:27:50.236 | }
08:27:50.274 | error Command failed with exit code 1.
08:27:50.275 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
08:27:50.284 | Error: Command "yarn run build" exited with 1
08:27:52.888 | Done with "package.json"
`

`
08:27:30.885 | $ node scripts/downloadDropboxContent.js && next build
-- | --
08:27:31.168 | (node:660) UnhandledPromiseRejectionWarning: DropboxResponseError: Response failed with a 409 code
08:27:31.168 | at /vercel/workpath0/node_modules/dropbox/cjs/src/response.js:34:11
08:27:31.168 | at processTicksAndRejections (internal/process/task_queues.js:93:5)
08:27:31.168 | at async main (/vercel/workpath0/scripts/downloadDropboxContent.js:27:15)
08:27:31.168 | (Use `node --trace-warnings ...` to show where the warning was created)
08:27:31.168 | (node:660) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
08:27:31.168 | (node:660) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`

package.json:

{
  "name": "personal-blog-template",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "node scripts/downloadDropboxContent.js && next dev",
    "build": "node scripts/downloadDropboxContent.js && next build",
    "start": "node scripts/downloadDropboxContent.js && next start"
  },
  "dependencies": {
    "@tailwindcss/typography": "^0.3.1",
    "autoprefixer": "^10.1.0",
    "cross-fetch": "^3.0.6",
    "date-fns": "^2.16.1",
    "dotenv": "^8.2.0",
    "dropbox": "^8.2.0",
    "gray-matter": "^4.0.2",
    "next": "10.0.4",
    "postcss": "^8.2.2",
    "react": "17.0.1",
    "react-dom": "17.0.1",
    "remark": "^13.0.0",
    "remark-html": "^13.0.1",
    "tailwindcss": "^2.0.2"
  },
  "devDependencies": {
    "@types/node": "^14.14.41",
    "@types/react": "^17.0.0",
    "cypress": "^7.1.0",
    "typescript": "^4.2.4"
  }
}

**Comment by rchrdchn** [comment]
Hey @LaurensKersbergen - I'm having the same issues trying to deploy my Next.js app using Vercel. I see you got your app up and working. Do you mind sharing how did you fix this, please?
